### DISCUSSION TITLE
How to add periodic breaks for vehicles?

### DISCUSSION THREAD

**Discussion by Arun0013** [google/or-tools#3160] [Routing (and legacy CP) questions] [answered]
I am solving a VRP (single-pickup (depot), multiple drop variant) with Time Windows. In the problem, I need to provide periodic breaks for all vehicles. For example, each vehicle should take a 30-minutes break after traveling every 3 hours. Additionally, the 3 hours traveling window should start only after visiting the first drop point i.e., the traveling time between the depot and the first drop point is not considered in the 3-hour travel window.

How to model this requirement using VRP ORTools?

**Comment by lperron** [comment, marked as answer]
Duplicate of the mailing list.
